test/e2e/app_deploy_test.go

```diff
@@ -0,0 +1,105 @@
+// Copyright 2023 VMware, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package e2e
+
+import (
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestAppDeploy(t *testing.T) {
```

Comment:
Should we add this test to [create_update_delete_test.go](https://github.com/vmware-tanzu/carvel-kapp/blob/develop/test/e2e/create_update_delete_test.go) with a more descriptive name?

Reply:
Yeah, it make more sense to move it here. Updated changes.